addons/index.md

```diff
@@ -7,16 +7,108 @@ layout: documentation
+| [Actions](actions.html)                 | Actions are predefined methods for openHAB rules and scripts                                                              |
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
+
+## Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
+Additional add-ons can be installed in the different ways, described below.
+
+### Through Paper UI
+
+Navigate to the add-ons section.
+Search for the desired add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/index_installation_paperui.gif)
+
+### By editing addons.cfg
+
+For this installation option you need to know the *id* of the desired add-on
+
+You can find it out in the [openHAB console]({{base}}/administration/console.html) with:
+
+```sh
+feature:list | grep ^openhab
+```
+
+This will give you a list of all available add-ons.
+It could probably look similar to this example:
+
+```text
+...
+openhab-transformation-xslt               | 0.9.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | XSLT Transformation
+openhab-voice-mactts                      | 0.9.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | MacOS Text-to-Speech
+openhab-binding-amazondashbutton          | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Amazon Dash Button Binding
+openhab-binding-astro                     | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Astro Binding
+openhab-binding-autelis                   | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Autelis Binding
+openhab-binding-avmfritz                  | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | AVM Fritz!Box Binding
+...
+openhab-binding-network                   │ 2.2.0            │          │ Uninstalled │ openhab-addons-2.2.0    │ Network Binding
+...
+```
+
+According to the [naming convention for bundles]({{base}}/administration/bundles.html#naming-convention-for-bundles) the *id* for our example is *network*.
+
+Another way of getting the correct id, is the url from the add-ons documentation page.
+For example the url for the [mqtt binding]({{base}}/addons/bindings/mqtt1/readme.html) is
+
+````text
+https://docs.openhab.org/addons/bindings/mqtt1/readme.html
```

Comment:
How about:

Another way of getting the correct ID is the URL of the add-on documentation page. To give an example, the URL for the [mqtt Binding's documentation]({{base}}/addons/bindings/mqtt1/readme.html) is [https://docs.openhab.org/addons/bindings/**mqtt1**/readme.html](), hence the ID would be "mqtt1" in this case.

Reply:
Or, consider, "Another way to find the correct ID is to look at the URL of the add-on documentation page.  For example, the URL for the mqtt Bindiing documentation is https://docs.openhab.org/addons/bindings/*mqtt1*/readme.html.  In this case, the ID would be "mqtt1".